### PR TITLE
updating to compiler v0.10.5

### PR DESCRIPTION
**Pull request by soupi**

### CODE REVIEW

File: src/Node/IRC.purs [hunk 1]
```diff
@@ -12,19 +12,17 @@ module Node.IRC
   , onChannelMessage
   ) where
 
-import Prelude
-import Control.Monad.Eff
-import Control.Monad.Eff.Console (log, print, CONSOLE())
+import Prelude (Unit, bind, const, flip, pure, unit, void, ($), (<<<), (<>))
```
Comment (hdgarrood) [inline src/Node/IRC.purs]: Could we have this one be the open import please?

File: src/Node/IRC/BareBones.purs [hunk 2]
```diff
@@ -9,8 +9,8 @@ module Node.IRC.BareBones
   , say
   ) where
 
-import Prelude
-import Control.Monad.Eff
+import Prelude (Unit)
```
Comment (hdgarrood) [inline src/Node/IRC/BareBones.purs]: Can this stay open too please?

File: bower.json [hunk 3]
```diff
@@ -1,6 +1,6 @@
 {
   "name": "purescript-node-irc",
-  "version": "1.0.0",
+  "version": "2.0.0",
```
Comment (hdgarrood) [inline bower.json]: Could you please just remove the version field actually?

File: docs/Node/IRC/BareBones.md [hunk 4]
```diff
@@ -3,13 +3,13 @@
 #### `IRC`
 
 ``` purescript
```
Comment (hdgarrood) [inline docs/Node/IRC/BareBones.md]: Could you please remove the docs folder from git?

**Review by hdgarrood — COMMENTED** [review]
Looks great, if you wouldn't mind addressing my comments and then I'll merge.

**Comment by soupi** [comment]
done

**Comment by hdgarrood** [comment]
Thanks!

**Pull request merged**